pytorch_toolkit/image_classification/init_venv.sh

```diff
@@ -18,7 +18,7 @@ fi
 
 # Download deep-object-reid
 git submodule update --init --recursive --recommend-shallow ../../external/deep-object-reid
-
+git submodule update --remote --merge ../../external/deep-object-reid
```

Comment:
to update the submodule to the latest commit. 

The problem was that after cloning the repository, the latest branch updates were not available. I thought this would fix this problem.

Reply:
You don't need to checkout the latest commit from submodule since it can be incompatible. Submodule is kinda link to a commit in an other repo. If you want to update submodule you should update this link and then on each recursive checkout this commit will be fetched.